HIP/hip-583.md

```diff
@@ -10,220 +10,452 @@ status: Accepted
 
-### Account Creation via alias only (Lazy Create)
+Since Hedera accounts require keys to ensure security, an `ECDSA` key matching the derived `evm address alias` must be
+provided in a future signed transaction issued by the owner of the account to take control of the account. Due to the
+nature of `ECDSA secp256k1` keys, the public key can be extracted from the signature and the signed bytes. It can then
+go through the derivation function to generate the associated `evm address`, which can be compared with the `alias` to
+prove the match and account ownership. This process will complete the creation of the account, taking it from a
+`Hollow Account` to a normal complete account with a valid public key.
 
-![lazy-create-with-address.png](../assets/hip-583/lazy-create-with-address.png)
+Note, in this case the final value of the account alias will remain as the `evm address alias` and not the `key alias`.
 
-The `CryptoCreate` transaction should be expanded to create a new account on the network when only a `public-address` alias is provided with no `public-key`.
+`EthereumTransaction` transactions where value is transferred  (i.e `tx.to` and `tx.value` are set) should also support
+this case. This will support users of `Ethereum-native wallets` and eth dev tools who are accustomed to creating and
+crediting an account's balance in one step. In this case the `tx.to` value of the RLP encoded transaction would dictate
+the `evm address alias` to be used.
 
-If a `public-key` is provided as an alias the `key` property on the account may be set automatically. When a `public-address` (based on an `ECDSA` key) is provided as an alias the `key` on the account will be left empty and only the `alias` set with a value of the `public-address`.  An account created in this manner will be known as a `Hollow Account` and will only be able to receive HBAR and applicable token transfers, but can not take part in transactions submitted by others requiring its signature.
+#### Proof of Ownership
 
-Since Hedera accounts require keys to ensure security, a matching `ECDSA` `public-key` must be provided in a future signed transaction issued by the owner of the `public-address`. Due to the nature of `ECDSA` keys the `public-key` and `public-address` may be extracted from the signature of a transaction signed by the `ECDSA` private key that owns the new account. This process will complete the creation of the account, taking it from a `Hollow Account` to a normal complete account with a valid `public-key`.
-Note, in this case the final value of the account alias will remain as the `public-address` and not the `public-key` as it would be in the `auto-create` case.
+If the `alias` is used during `CryptoTransfer` as the _target_ of the transfer, then there is no need to prove that
+anyone controls that account. If an `alias` is used in the `AccountID` of the payer, then ownership must be asserted,
+just as is done for any other transaction today.
```

Comment:
Add some text to indicate proof of ownership is required if you are the source of the transfer, not just the payer.